### ISSUE TITLE
extra language library library pack

### ISSUE THREAD

**Issue by lianli34312** [dhrone/pydPiper#67]
I am currently using 0.96" OLED for Moode. But it shows "???" when dealing with japanese/chinese songs. It there any solutions to add extra language library library pack?

**Comment by lianli34312** [comment]
`FONTS = {
	'small': { 'default':True, 'file':'latin1_5x8.fnt','size':(10,16) }, 
#'large': { 'file':'BigFont_10x16.fnt', 'size':(20,30) },
	'large': { 'file':'Vintl01_10x16.fnt', 'size':(20,30) },
	'tiny': { 'file':'upperasciiwide_3x5_fixed.fnt', 'size':(5,5) },
}

TRUETYPE_FONTS = {
	'iming28': { 'file':'/home/pi/pydPiper/ttf/iming.ttf', 'size':28 }
}
`

Still didn't work. Do I need to change the code inside `FONTS ` also?

**Comment by dhrone** [comment]
No.  Truetype fonts are only specified within the TRUETYPE_FONTS section of the pages file.  You do need to make sure that you are specifying the fonts you define within the TRUETYPE_FONTS section within the widgets section as well...

'''
TRUETYPE_FONTS = {
  'ttsmall': { 'file':'/home/pi/pydPiper/ttf/iming.ttf', 'size':8 }
  'ttlarge': { 'file':'/home/pi/pydPiper/ttf/iming.ttf', 'size':12 }
}

WIDGETS {
	'artist': { 'type':'ttext', 'format':'{0}', 'variables':['artist'], 'font':'ttlarge','varwidth':True,'effect':('scroll','left',1,1,20,'onloop',3,125)},
	'title': { 'type':'ttext', 'format':'{0}', 'variables':['title'], 'font':'ttsmall','varwidth':True,'effect':('scroll','left',1,1,20,'onloop',3,125)},
}
'''

Notice that the type for the widgets that are using the truetype font are ttext, not text which is used for the low resolution fonts necessary for typical LCD displays.
